frontend/src/app/api/OpportunityListingAPI.ts

```diff
@@ -7,7 +7,7 @@ import BaseApi from "./BaseApi";
 
 export default class OpportunityListingAPI extends BaseApi {
   get version(): string {
-    return "v1";
+    return "v0.1";
```

Comment:
I'm not sure if you want to switch this - as soon as the API deploys are fixed, you'll want to be on v1 as several fields are missing on v0.1.

Reply:
@chouinar misunderstood what you said in slack. Updating.